packages/loader/container-loader/src/container.ts

```diff
@@ -1629,10 +1615,14 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         }
 
         const state = this.connectionState === ConnectionState.Connected;
+
+        assert(this.context !== undefined,
```

Comment:
you will never hit this assert, if this._context is undefined, this.context will throw as it never returns undefined